form/validation_group_service_resolver.rst

```diff
@@ -1,5 +1,5 @@
-How to Dynamically Configure Validation Groups
-==============================================
+How to Dynamically Configure Form Validation Groups
```

Comment:
@xabbuh when the main title of an article changes, should we add a label for the previous title?